jac-uk/apply
# Bugfix/TS 341 ASC jump and edit in review

## What's included?
Closes https://github.com/jac-uk/ticketing-system/issues/341

- When you edit ASC, the page should not jump around.
- Add the change link to ASC on the review page.

**Note:** Since the ASC and self assessment upload are on the same page, we might need to rethink the UI and flow on the "Self assessment competencies" page.

## Who should test?
✅ Product owner
✅ Developers

## How to test?

[Example vacancy](https://jac-apply-develop--pr1208-fix-ts-341-asc-jump-xpdoava2.web.app/vacancy/x3CJhzZr1MCMMPUdMw93/)

Steps:
1. Apply for the vacancy.
2. Edit ASC on the "Self assessment competencies" page and check if the page doesn't jump around.
3. Go to the review page and check if there is a "Change" link next to "Additional Selection Criteria" that allows you to edit ASC.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

Demo of preventing from jumping around when you edit ASC:

[video-30275792-54dd8ba4ee350b04cb2b5bd64a1a510e.webm](https://github.com/user-attachments/assets/2aa83435-7508-4012-b6cd-d38ea608363e)

Demo of adding a link to edit ASC

https://github.com/user-attachments/assets/159f96ab-ac60-420e-a8b9-484c089c20a6

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
